tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -342,13 +380,18 @@ module.exports = {
      * @param {string} enableOrDisable
      * @param {string} path
      */
-    toggleFileOrFolderCheckbox: function (enableOrDisable, path) {
+    toggleFileOrFolderCheckbox: async function (enableOrDisable, path) {
+      await this.filesListScrollToTop()
+      await this.waitForFileVisible(path)
+
       const fileCheckbox = this.getFileRowSelectorByFileName(path) +
         this.elements.checkboxInFileRow.selector
 
+      await this.toggleCheckbox(enableOrDisable, fileCheckbox, 'xpath')
+
+      await this.pause(500)
```

Comment:
lets not do fix-value sleeps, is there some visual feedback that we can wait for?

Reply:
Leftover from debugging. Will remove.